storage/ast/bolt_listener_test.go

```diff
@@ -604,6 +604,9 @@ func TestSetFunctions(t *testing.T) {
 
 		{"not isEmpty linkIds, true", "not isEmpty(link.ids)", true},
 		{"not isEmpty lunkIds, false", "not isEmpty(lunk.ids)", false},
+
+		{"and set ops, true", "not isEmpty(link.ids) and count(link.ids) = 3", true},
+		// TODO: FIX THIS CASE {"and set ops explicit equals, true", "isEmpty(link.ids) = false and count(link.ids) = 3", true},
```

Comment:
IsEmpty(set-symbol) = <bool const> is currently broken. Fixing it requires grammar changes, and the cost for that is currently high b/c of antlr bugs. Going to put this off till later as I don't think it's high priority.